sway/sway-ipc.7.scd

```diff
@@ -1131,6 +1131,9 @@ following properties:
 |- xkb_active_layout_index
 :  integer
 :  (Only keyboards) The index of the active keyboard layout in use
+|- scroll_factor
+:  floating
+:  Changes the scroll factor for the specified input device.
```

Comment:
The description reads a little bit weird. Maybe something among these lines would be better?

```
(Only pointers) Multiplier applied on scroll event values.
```

Reply:
Isn't scroll_factor also applied to touchpads? Maybe (Only pointers and touchpads)

Reply:
A touchpad is a pointer.